### DISCUSSION TITLE
Issue 1247

### DISCUSSION THREAD

**Discussion by 0xArch3r** [opentdf/platform#1262] [General]
@strantalis I saw your question but the thread for the issue was locked so I couldnt respond. We have a pretty robust policy with over 260 different attributes and a complex subject mapping. When trying to call functions like GetEntitlements with no scope, or even trying to chunk those requests we seem to hit a message size limit due based on the logging. When we set that size higher it does work.

I made a PR that allows the viper config to obey the json tags on the config structs so it is less misleading.

**Comment by strantalis** [comment]
@0xArch3r  Sorry about that. I unlocked the issue. We hit the same issue as well and have a resolution for now we think. But we also know the solution will need to continue to evolve overtime. 

https://github.com/opentdf/platform/pull/1271

**Reply by jrschumacher** [reply]
@0xArch3r That's my fault. Sorry to create extra work for you.

**Reply by 0xArch3r** [reply]
@jrschumacher no worries!
